pkg/virt-operator/resource/apply/delete_test.go

```diff
@@ -169,6 +174,73 @@ var _ = Describe("Deletion", func() {
 			Expect(needRemoved).To(BeEmpty())
 		})
 	})
+
+	Context("Node label deletion", func() {
+		var clientset *fake.Clientset
+		var kubevirtClient *kubecli.MockKubevirtClient
+		var ctrl *gomock.Controller
+
+		BeforeEach(func() {
+			ctrl = gomock.NewController(GinkgoT())
+			clientset = fake.NewSimpleClientset()
+			kubevirtClient = kubecli.NewMockKubevirtClient(ctrl)
+			kubevirtClient.EXPECT().CoreV1().Return(clientset.CoreV1()).AnyTimes()
+		})
+
+		AfterEach(func() {
+			ctrl.Finish()
+		})
+
+		It("should delete kubevirt labels from nodes", func() {
+			nodes := &v1.NodeList{
```

Comment:
```
pkg/virt-operator/resource/apply/delete_test.go:195:17: undefined: v1.NodeList
pkg/virt-operator/resource/apply/delete_test.go:196:17: undefined: v1.Node
```
You need to check your imports.
Please, always run `make`, `make generate`, and `make test` before pushing, so that you are sure it works on your local machine. Thanks

Reply:
I don't know what's going on there in this case, but it is related to your local machine. It looks like a wrong project configuration

Reply:
@fossedihelm now , every thing is fine .please test it .

Reply:
@anishbista60 The only [Change](https://github.com/kubevirt/kubevirt/compare/82df0cc24d6daebebf57c1e5c5c284bd1b5c3133..9c4d92527d0b1d04575be0fcfa6cbf201df69f43) I see it related to a label string. 
The problem is that you are using `v1` as `v1 "k8s.io/api/core/v1"` but in the delete_test file is `v1 "kubevirt.io/api/core/v1"`.
I think you need to import `k8s.io/api/core/v1` maybe as (k8sv1 "k8s.io/api/core/v1") and use it `nodes := &k8sv1.NodeList{`.

But, you said that everything is fine now, are you sure you pushed all the changes? Thanks

Reply:
@fossedihelm  Ok. Thanks

Reply:
This is not compiling in my local machine. You have to do the same for the `v1.Node` -> `k8sv1.Node`. 
Doesn't your `make && make generate` return any errors?